### ISSUE TITLE
[Upgrade] Bootstrap v4

### ISSUE THREAD

**Issue by maxeckel** [EazyServer/Vue-Bootstrap-Calendar#4]
Hello Yarob,

what do you think about a Branch using Bootstrap 4?

I already started the work in my Fork and would like to merge it when it's ready.

**Comment by EazyServer** [comment]
Hi Max,

It would be great to support B4.
But in this package I am only using class names from B3, is that going to change on B4?

Thanks

**Comment by maxeckel** [comment]
Yes, in Bootstrap 4 some things/components have changed. For example, panels have been replaced by cards.

So we would have to publish another package for bootstrap 4 support (for example vue-bootstrap4-calendar)

Another option would be to consider version 1.* to be Bootstrap 3 and add an 2.* version for Bootstrap 4

The Version in my Branch is already fully functional and I integrated an add event modal and fixed the i18n issue.

Later this Day i'll integrate these changes into Master and create new pull requests for that.

**Comment by EazyServer** [comment]
Awesome work! will have a look once you'r finished.

**Comment by maxeckel** [comment]
The transition from B3 to B4 is already finished.

**Comment by claudeberube** [comment]
I am an issue using the latest template webpack.  [Vue warn]: Failed to mount component: template or render function not defined.
I installed the deafult Runtime + Compiler option.  So, I wonder If anybody can replicate it.
